src/RadhydroShockCGS/test_radhydro_shock_cgs.cpp

```diff
@@ -25,7 +25,7 @@ struct ShockProblem {
 constexpr double a_rad = 7.5646e-15;		  // erg cm^-3 K^-4
 constexpr double c = 2.99792458e10;		  // cm s^-1
 constexpr double k_B = C::k_B;			  // erg K^-1
-constexpr double m_H = quokka::hydrogen_mass_cgs; // mass of hydrogen atom [g]
+constexpr double m_H = C::m_u; // atomic mass unit
```

Comment:
For clarity, this should probably be removed, and replaced with $C::m_u$ directly wherever it's used.